pyproject.toml

```diff
@@ -23,25 +23,34 @@ dependencies = [
     "PyYAML>=6.0.2"
 ]
 
-[project.urls]
-Homepage = "https://github.com/ambiqai/neuralSPOT"
-Documentation = "https://ambiqai.github.io/neuralSPOT"
-Repository = "https://github.com/ambiqai/neuralSPOT"
-
-
 [tool.setuptools]
 packages=[
     "neuralspot",
     "neuralspot.rpc",
     "neuralspot.rpc.GenericDataOperations_EvbToPc",
     "neuralspot.rpc.GenericDataOperations_PcToEvb",
+    "neuralspot.tools",
 ]
 
 [tool.setuptools.package-dir]
-neuralspot = "neuralspot"
+neuralspot = ""
 "neuralspot.rpc" = "neuralspot/ns-rpc/python/ns-rpc-genericdata"
 "neuralspot.rpc.GenericDataOperations_EvbToPc" = "neuralspot/ns-rpc/python/ns-rpc-genericdata/GenericDataOperations_EvbToPc"
 "neuralspot.rpc.GenericDataOperations_PcToEvb" = "neuralspot/ns-rpc/python/ns-rpc-genericdata/GenericDataOperations_PcToEvb"
+"neuralspot.tools" = "tools"
```

Comment:
Do we need to add utils and autodeploy here too? I personally am not familiar with what packages-dir does.